<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预报</string>
  </property>
  <widget class="QWidget" name="centralwidget_main">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_date">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>公历</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_NanYang">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>南阳市</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_low_tmp">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>最低温度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_airs">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>空气质量</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_clothes">
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="plainText">
           <string>穿衣指数</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_weekday">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>星期</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_weather">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>天气</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_heigh_tmp">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>最高温度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_rays">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
            </property>
            <property name="text">
             <string>紫外线强度</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_remind">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10000</width>
            <height>36</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="tabChangesFocus">
           <bool>false</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="plainText">
           <string>请输入温馨提醒</string>
          </property>
          <property name="tabStopWidth">
           <number>80</number>
          </property>
          <property name="backgroundVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="button_getdocument">
        <property name="text">
         <string>获取稿件</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_statue">
            <property name="text">
             <string>状态</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_state_did">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="citysWeatherLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget_citysWeather">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="saveSite_action"/>
    <addaction name="lookingFiles_action"/>
    <addaction name="separator"/>
    <addaction name="about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="saveSite_action">
   <property name="text">
    <string>选择保存位置</string>
   </property>
  </action>
  <action name="lookingFiles_action">
   <property name="text">
    <string>查看文件归档</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
